sqlmypy.py

```diff
@@ -386,6 +394,52 @@ class User(Base):
                     column=ctx.default_return_type.column)
 
 
+def session_query_hook(ctx: MethodContext) -> Type:
+    """
+    Turns session.query(...) into typed Query.
+
+    Examples:
+
+        session.query(User) -> Query[User]
+        session.query(User, Order) -> Query[Tuple[User, Order]]
+        session.query(User, User.id) -> Query[Tuple[User, int]]
+        session.query(User.id) -> Query[Tuple[int]]
+    """
+    # TODO take a look at Session.query_cls? Do we have to do this with generics?
+
+    cnt_entities = 0
+
+    def map_arg(arg: Type) -> Type:
+        nonlocal cnt_entities
+        # A model class (well, in this case a callable constructor)
+        # Example:
+        # session.query(Employee) -> Query[Employee]
+        if isinstance(arg, CallableType) and arg.is_type_obj():
+            cnt_entities += 1
+            return arg.ret_type
```

Comment:
This will also add an item for an unrelated class, e.g. `session.query(int, str)`. I think we should only add model classes here (I think there is already metadata for this on `TypeInfo`). Should we also give an error for random non-model classes?

Reply:
This wold be a problem if someone uses classical mapping